code/modules/mob/living/silicon/ai/multicam.dm

```diff
@@ -168,7 +168,7 @@ GLOBAL_DATUM(ai_camera_room_landmark, /obj/effect/landmark/ai_multicam_room)
 		var/datum/camerachunk/CC = VV
 		for(var/V in CC.cameras)
 			var/obj/machinery/camera/C = V
-			if(!C.can_use() || (get_dist(C, src) > telegraph_range))
+			if(QDELETED(C) || !C.can_use() || (get_dist(C, src) > telegraph_range))
```

Comment:
I know you won't like it, but this is a good case for a `stack_trace()`.
```
/obj/machinery/camera/Destroy()
	GLOB.cameranet.cameras -= src
```
This exists and means there should be no `QDELETED` cameras in the chunk. Which makes me suspect there's a `null` entry or something.

Let me know if you are unwilling, and I'll do a PR myself to hunt it.

Reply:
I am very unwilling since this is not an issue. Do you understand the entirety of camera code? Are you 100% sure this is an issue instead of me just forgetting to add this check during porting? Check the rest of the code, qdeleted is checked regularly. Are you willing to refactor the entirety of cameranet code so this is not necesary?